install.sh

```diff
@@ -302,13 +303,26 @@ nvm_do_install() {
     echo "   OR"
     echo "=> Append the following lines to the correct file yourself:"
     command printf "${SOURCE_STR}"
+    echo "=> Please also append the following lines to the if you are using bash/zsh shell:"
```

Comment:
i'd like this line to only be output when the user *is* using them - in other words, in the same circumstances where "Appending bash_completion source string" happens.